extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -175,7 +175,10 @@ private function getTopContributorsDetails( Array $ids ) {
 	 * @return array
 	 */
 	private function getNavigationData() {
-		return $this->sendRequest( 'NavigationApi', 'getData' )->getData();
+		//getData always return array with navigation and wiki keys.
+		//See: includes/wikia/api/NavigationApiController.class.php:28
+		//and: includes/wikia/models/NavigationModel.class.php:175
+		return $this->sendRequest( 'NavigationApi', 'getData' )->getData()['navigation']['wiki'];
```

Comment:
well code is straightforward and if some fatal error occur along the way it won't work anyway.
If this change is going to be more than this one liner here and in Mercury (your comment about caching) then I'm not sure if it's worth doing at this point.
We already decided we need to rethink local nav in Mercury.
Maybe we should fix it then?